diffkemp/simpll/Utils.cpp

```diff
@@ -874,3 +875,66 @@ StructType *getTypeByName(const Module &Mod, StringRef Name) {
     return Mod.getTypeByName(Name);
 #endif
 }
+
+/// Given an instruction and a pointer value, try to determine whether the
+/// instruction may store to the memory pointed to by the pointer. This is
+/// currently implemented only as a set of heuristics - if they do not suffice,
+/// we return true.
```

Comment:
Yet there's `return false` at the end of the function :-)

Reply:
:) Reworded.